packages/apps/kadena-docs/src/scripts/remarkHeadersToProps.js

```diff
@@ -0,0 +1,66 @@
+import { toString } from 'mdast-util-to-string';
+
+const getTagName = (depth = 1) => `h${depth}`;
+
+const getParentHeading = (parent, newChild) => {
+  const orderArr = ['h1', 'h2', 'h3', 'h4', 'h5', 'h6'];
+  const nodes = [parent];
+  let child = parent.children[parent.children.length - 1] ?? parent;
+
+  while (child.depth < newChild.depth) {
+    const getLastChild = child.children[child.children.length - 1];
```

Comment:
this is now named like a `getter` but you don't really get it, it's just a reference right? So shouldn't this be renamed to `lastChild`?

Reply:
renamed it to lastHeading